docs/intro.rst

```diff
@@ -0,0 +1,7 @@
+===============
+Getting started
+===============
+
+TODO: Write about how to include Resolwe in a Django project and explain
+settings parameters. Create an example Django project in the ``docs/example``
```

Comment:
Here I would have a minimal project. I would have here a proper project (what the django-admin.py create project gives you). With the front-end you could play with and so on.

I don't know, but maybe I do not wish to clutter the test project with examples for the documentation.